### DISCUSSION TITLE
How to make W11's snipping tool work normally?

### DISCUSSION THREAD

**Discussion by notNSANE** [eythaann/Seelen-UI#150] [General]
Say that I launch the snipping tool via a keyboard press like PrntScrn, right now, after I snip, nothing happens. 

I know there's something inside Seelen to handle this but I don't  understand how it works.

**Comment by eythaann** [comment]
I use snipping tool regular but I thought PrntScrn print the entire screen cuz it detects Seelen as a fullscreen app, btw using `Win + Shift + S` is a better shortcut to open the snipping tool.

Note: currently if Window Manager is `on` there is a shortcut that overrides snipping tool, I will change the default value for new users to solve this but this has to be manually changed by the rest of users.

**Reply by notNSANE** [reply]
you mean Tiling Window Manager toggle? It's off and it still doesn't work for me.

Snipping Tool for images has four types of capture (Rectangle, Window, Fullscreen and Freeform). I use Rectangle 99% of the time. 

I tried using Fullscreen and it still doesn't work. I killed two apps for my mouse and it still doesn't work... I'll try restarting everything since I have one windows update anyways.


BTW I'm using Seleen UI 2.

Edit: Still not working :(

**Reply by eythaann** [reply]
did you try `Win + Shift + S`? cuz it opens the snipping tool to capture on any mode. what exactly isn't working for you? if is the notification there is a reported bug for that: #55

**Reply by notNSANE** [reply]
it was the notification indeed! That's what wasn't working, thanks for linking the bug #.
